notebooks/calculateSharpness.ipynb

```diff
@@ -0,0 +1,929 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "# QC script to calculate sharpness metric for images in a plate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "The following script will access the IDR images in a facility manager's context,\n",
+    "\n",
+    "The QC script does the following,\n",
+    "\n",
+    "1. Extracts Images from OMERO (in this case IDR)\n",
+    "2. Calcualtes sharpness based on the algorithm defined in the following publication,\n",
+    "  * [Image Sharpness Measure for Blurred Images in Frequency Domain] (https://doi.org/10.1016/j.proeng.2013.09.086)\n",
```

Comment:
this doesn't seem to work as an external link -- maybe because of space within "] ("?